apps/meteor/tests/end-to-end/api/01-users.js

```diff
@@ -3228,7 +2853,14 @@ describe('[Users]', function () {
 			});
 		});
 		describe('unsuccessful cases', () => {
-			it('Remove necessary permission "create-personal-accss-tokens" to user', () => updatePermission('create-personal-access-tokens', []));
+			before(async () => {
```

Comment:
we can remove the async and the await, like here: https://github.com/RocketChat/Rocket.Chat/pull/31597/files#r1539137621